### PR TITLE
NIAD-50: env var parser fix

### PR DESCRIPTION
**Pull request by bartek-sarul**

### CODE REVIEW

**Review by Broom94 — DISMISSED** [review]

File: mhs/outbound/main.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def main():
                                                   ca_certs=secrets.get_secret_config('CA_CERTS'))
     max_retries = int(config.get_config('OUTBOUND_TRANSMISSION_MAX_RETRIES', default="3"))
     retry_delay = int(config.get_config('OUTBOUND_TRANSMISSION_RETRY_DELAY', default="100"))
-    validate_cert = bool(config.get_config('OUTBOUND_VALIDATE_CERTIFICATE', default="True"))
+    validate_cert = config.get_config('OUTBOUND_VALIDATE_CERTIFICATE', default=str(True)).lower() == str(True).lower()
```
Comment (grzegorz-robak) [inline mhs/outbound/main.py]: If env var is set then `config.get_config()` will return String. I think that isn't the way it should be

**Review by grzegorz-robak — CHANGES_REQUESTED** [review]

File: mhs/outbound/main.py (hunk 1 above)
Comment (mattd-kainos) [inline mhs/outbound/main.py]: `default='true').lower() == 'true'` would be simpler. Is there a reason its written this way?

Comment (grzegorz-robak) [reply mhs/outbound/main.py]: ok, as I read it I saw `default=str(True).lower()`. Looks like it's easy to misread it. Please split that into two lines

**Review by grzegorz-robak — CHANGES_REQUESTED** [review]

File: mhs/outbound/main.py (hunk 1 above)
Comment (bartek-sarul) [reply mhs/outbound/main.py]: just to omit potential error in typing a string
anyway, I've added a str2bool function

**Review by grzegorz-robak — DISMISSED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -152,6 +152,17 @@ pipeline {
                     }
                     post {
                         always {
+                            sh label: 'Docker status', script: 'docker ps -al'
+                            sh label: 'Dump container logs', script: '''
+                                docker logs ${BUILD_TAG_LOWER}_route_1 > logs/route.log
```
Comment (grzegorz-robak) [inline Jenkinsfile:158]: I think that component tests doesn't use route, but fake spineroutelookup

**Review by grzegorz-robak — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 2 above)
Comment (bartek-sarul) [reply Jenkinsfile:158]: it's just how the container is named

**Review by petervdm — APPROVED** [review]

**Review by grzegorz-robak — APPROVED** [review]

**Pull request merged**